doc/tutorial.txt

```diff
@@ -232,6 +217,45 @@ below are equals:
 	}
 }
 
+*** thread and phase object properties ***
+
+Several properties can be set at thread and/or phase levels. All these
+properties are optional and default value will be used if nothing is defined.
+
+* loop: Integer. Define the number of times the parent object must be run.  The
+  parent object can be a phase or a thread. For phase object, the loop defines
+the number of time the phase will be executed before starting the next phase.
+For thread object, the loop defines the number of times that the complete
+"phases" object will be executed. The default value is -1 for thread object and
+1 for phases.
+
+*** scheduling policy
```

Comment:
Do you think it might be better to create an Object for policy data?
It would contain policy name, priority and deadline parameters.
Similar to what we have for timers.